### DISCUSSION TITLE
Recomendación de extensiones VS Code

### DISCUSSION THREAD

**Discussion by eldhaim** [Launch-X-Latam/MisionFrontEnd#58] [Ideas]
Para aprendizaje de HTML-CSS-JAVASCRIPT son ideales para generar un entorno sencillo de aprendizaje básico y de forma local.

![image](https://user-images.githubusercontent.com/86509650/192836312-2e1118c6-2da9-48af-8764-ecd90b14277f.png)
![image](https://user-images.githubusercontent.com/86509650/192836424-72878dd8-6ca4-4e73-865d-bf8fb32f1cca.png)

**Comment by JorgeDevmm** [comment]
Excelente recomendaciones, agregaría el de 
captura de código
![image](https://user-images.githubusercontent.com/49603096/192843694-e43e001f-bd8c-4381-a82f-9c4f6cbdb3f8.png)
iconos
![image](https://user-images.githubusercontent.com/49603096/192843755-84b07587-d473-44f0-b36e-6039f1006e5b.png)
barra de tareas inferior
![image](https://user-images.githubusercontent.com/49603096/192843853-6ed25d11-0cff-4529-9074-29deb29b820b.png)

**Comment by renzocastro** [comment]
Para tener control de la indentación (Tab/space)
![image](https://user-images.githubusercontent.com/206445/193221550-c5fbf385-9a62-4a01-b143-ccfa9cc3cb4f.png)

Más poder al gestor de versionado de código en git:
![image](https://user-images.githubusercontent.com/206445/193221789-f56bb43a-25fa-4833-a8ef-5e29b72aa3ad.png)
